config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -124,7 +123,6 @@ if (CONFIG_NORDIC_SECURITY_BACKEND)
 
     list(APPEND CHIP_CFLAGS
         -DMBEDTLS_CONFIG_FILE=<nrf-config.h>
-        -DMBEDTLS_PK_WRITE_C
         -DMBEDTLS_X509_CREATE_C
```

Comment:
Have you tried to replace this as well?

https://github.com/nrfconnect/sdk-nrfxlib/blob/master/nrf_security/Kconfig#L1643

Reply:
It worked